pythonforandroid/recipes/hostpython2/__init__.py

```diff
@@ -1,17 +1,22 @@
 
 from pythonforandroid.toolchain import Recipe, shprint, current_directory, info, warning
 from os.path import join, exists
-from os import chdir
+from os import environ
 import sh
 
 
 class Hostpython2Recipe(Recipe):
-    version = '2.7.2'
-    url = 'http://python.org/ftp/python/{version}/Python-{version}.tar.bz2'
+    version = '2.7.9'
+    url = 'http://python.org/ftp/python/{version}/Python-{version}.tgz'  # tar.bz2'
```

Comment:
Why is this changed to `.tgz`? Are the bzip2 archives not distributed anymore? bzip2 should be preferred as it is a smaller download at the expense of slower extraction - but people are far more likely to have download caps than execution caps.